packages/paste-website/src/components/shortcodes/pattern-header/index.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Heading} from '@twilio-paste/heading';
+import {Text} from '@twilio-paste/text';
+import {Stack} from '@twilio-paste/stack';
+import {SuccessIcon} from '@twilio-paste/icons/esm/SuccessIcon';
+import {P} from '../../Typography';
+import {Breadcrumb, BreadcrumbItem} from '../../breadcrumb';
+import {FigmaStatus} from '../../component-status/FigmaStatus';
+import {PackageStatus} from '../../component-status/PackageStatus';
+import {PeerReviewStatus} from '../../component-status/PeerReviewStatus';
+
+const PatternHeaderBasic: React.FC<{
+  name: string;
+}> = ({name}) => (
+  <>
+    <Breadcrumb>
+      <BreadcrumbItem to="/">Home</BreadcrumbItem>
+      <BreadcrumbItem to="/patterns">Patterns</BreadcrumbItem>
+    </Breadcrumb>
+    <Heading as="h1" variant="heading10">
+      {name}
+    </Heading>
+  </>
+);
+
+interface PatternHeaderProps {
+  description: string;
+  name: string;
+  patternStatus?: [
+    {
+      node: {
+        data: {
+          Documentation: boolean;
+          Figma: string;
+          Design_committee_review: string;
+          Engineer_committee_review: string;
+          Code: string;
+        };
+      };
+    }
+  ];
+}
+
+const PatternHeader: React.FC<PatternHeaderProps> = ({description, name, patternStatus}) => {
+  if (patternStatus == null || patternStatus[0] == null || patternStatus[0].node == null) {
+    return <PatternHeaderBasic name={name} />;
+  }
+
+  const {
+    Figma,
+    Documentation,
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    Design_committee_review,
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    Engineer_committee_review,
+    Code,
+  } = patternStatus[0].node.data;
+
+  return (
+    <>
+      <PatternHeaderBasic name={name} />
+      <P variant="lead">{description}</P>
+      <Box marginBottom="space100">
```

Comment:
The line of icons are shared between pattern and component headers, is there opportunity to share the code between them too?

Reply:
Yeah. Not thats one component.